src/platform/testing/e2e/cypress/support/commands/injectAxeThenAxeCheck.js

```diff
@@ -1,16 +1,7 @@
 /**
  * Combines two common, sequentially called functions.
  */
-Cypress.Commands.add('injectAxeThenAxeCheck', (context, tempOptions) => {
```

Comment:
`injectAxeThenAxeCheck` was never called with an args, so it's safe to change the function args and make it a clean pass-through to `axeCheck()`